bin/ch/WScriptJsrt.cpp

```diff
@@ -981,6 +982,8 @@ JsValueRef __stdcall WScriptJsrt::LoadBinaryFileCallback(JsValueRef callee,
                         returnValue = arrayBuffer;
                     }
                 }
+ErrorStillFree:
+                HeapFree(GetProcessHeap(), 0, (void*)fileContent);
```

Comment:
> Why are we using free on one and HeapFree on the other ?

Because one is malloc'd and one is HeapAlloc'd. I didn't want to make more changes than immediately necessary.

> Can we just make fileContent an AutoString ?

Not without changing how LoadScriptFromFile works; there's no good way to have an AutoString get externally initialized and set all of the properties correctly (e.g. length).